src/Employer/Employer.Web/ViewModels/Part1/TitleVacancy/TitleViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using Esfa.Recruit.Employer.Web.ViewModels.Validations;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Esfa.Recruit.Employer.Web.ViewModels.Part1.TitleVacancy
+{
+    public class TitleViewModel
+    {
+        [Required]
+        public string Title { get; set; }
```

Comment:
What about the validation attributes `[StringLength(100, MinimumLength = 1)]` as in the story.

Reply:
The story was updated after my PR. I'll revisit now